plenum/server/replica.py

```diff
@@ -2312,16 +2315,16 @@ def dequeue_prepares(self, viewNo: int, ppSeqNo: int):
             while self.preparesWaitingForPrePrepare[key]:
                 prepare, sender = self.preparesWaitingForPrePrepare[
                     key].popleft()
-                self.logger.debug("{} popping stashed PREPARE{}".format(self, key))
+                self.logger.info("{} popping stashed PREPARE{}".format(self, key))
                 self.threePhaseRouter.handleSync((prepare, sender))
                 i += 1
             self.preparesWaitingForPrePrepare.pop(key)
-            self.logger.debug("{} processed {} PREPAREs waiting for PRE-PREPARE for"
-                              " view no {} and seq no {}".format(self, i, viewNo, ppSeqNo))
+            self.logger.info("{} processed {} PREPAREs waiting for PRE-PREPARE for"
+                             " view no {} and seq no {}".format(self, i, viewNo, ppSeqNo))
 
     def enqueue_commit(self, request: Commit, sender: str):
-        self.logger.debug("Queueing commit due to unavailability of PREPARE. "
-                          "Request {} from {}".format(request, sender))
+        self.logger.info("Queueing commit due to unavailability of PREPARE. "
+                         "Request {} from {}".format(request, sender))
```

Comment:
I think, that we shoud add "key" into log string, for checking equality of key and value in the commitsWaitingForPrepare

Reply:
Also, please add the same logs into dequeue_commit and enqueue_commit method too.